lib/utils/deprecate.js

```diff
@@ -0,0 +1,47 @@
+var Logger = require('../logger');
+var stackTrace = require('stack-trace');
+var colorize = require('../ui/colorize');
+
+var logger = new Logger();
+
+exports.setMustShowWarnings = function (mustShowWarnings) {
+    if (mustShowWarnings) {
+        logger.showWarnings();
+    } else {
+        logger.hideWarnings();
+    }
+};
+
+exports.deprecateModule = function (module, removeVer, replacement) {
```

Comment:
параметры лучше передать объектом, тогда в месте использования будет намного легче читать:
```js
deprecate({module: 'test-logger', since: 'v1.0.0', replacedWith: 'mock-enb package'});
```

ну и соответственно у тебя будет экспортиться одна функция, а `method` или `module` будет решаться по свойству